frontend/src/scenes/apps/frontendAppsLogic.tsx

```diff
@@ -31,7 +31,7 @@ export const frontendAppsLogic = kea<frontendAppsLogicType>([
             loadFrontendApp: async ({ id, pluginId, reload, attempt }) => {
                 if (!values.appConfigs[id]) {
                     if (pluginsLogic.findMounted()) {
-                        const pluginConfig = Object.values(pluginsLogic.values.pluginConfigs).find((c) => c.id === id)
+                        const pluginConfig: PluginConfigType | undefined = pluginsLogic.values.getPluginConfig(id)
```

Comment:
why do I need to call out the type to not get type errors (see commit 1)

Reply:
I'm honestly not sure. It should work, but doesn't. 🤷 

<img width="594" alt="image" src="https://github.com/PostHog/posthog/assets/53387/328844de-e60c-48e5-a8dd-6f8d758e8763">

<img width="638" alt="image" src="https://github.com/PostHog/posthog/assets/53387/9cd8a4b6-a47a-4d5e-8fce-d3eb2fb91d1d">